yb-voyager/src/query/queryissue/issues_ddl.go

```diff
@@ -523,7 +523,7 @@ func NewPathDatatypeIssue(objectType string, objectName string, sqlStatement str
 var polygonDatatypeIssue = issue.Issue{
 	Type:        POLYGON_DATATYPE,
 	Name:        "Unsupported datatype for Live migration - Polygon",
```

Comment:
for Issue.Name lets maintain a variable in `constants.go` file of this package.
I think we are doing that for most of the issues, already. ?

Reply:
Done